### DISCUSSION TITLE
Display plötzlich tot

### DISCUSSION THREAD

**Discussion by dajo77** [tbnobody/OpenDTU#1284] [General]
Ich habe an meiner OpenDTU ein 1,3 " Display (SH1106) angeschlossen, welches nach der Neueinrichtung auch problemlos funktioniert hat.  Heute, nachdem ich die OpenDTU Software aktualisiert habe, bleibt das Display tot.
Ich habe  die alten .json Dateien neu hochgeladen und natürlich das richtige Profil gewählt; das Display bleibt tot.

Jemand eine Idee, was ich noch probieren kann ?
Ein anderes Display habe ich leider nicht zum Testen.

**Comment by JorisZzz** [comment]
I have the same issue. I have the breakout board from Blinkyparts:
https://shop.blinkyparts.com/en/OpenDTU-Your-evaluation-for-your-balcony-solar-system/blink237542
It's got the WROOM ESP32 and a 1.3" IIC display. The display worked fine, until I flashed the device (don't ask why -.-). Also the LED's don't work anymore.
To flash, I first erased the memory, then flashed opendtu-generic_esp32.factory.bin, and then opendtu-generic_esp32.bin (v23.8.28)

**Reply by tbnobody** [reply]
> To flash, I first erased the memory, then flashed opendtu-generic_esp32.factory.bin, and then opendtu-generic_esp32.bin (v23.8.28)

It is enough to flash the `opendtu-generic_esp32.factory.bin` if you flashing via USB. But in this case you also erased all your configuration and device profiles. (this will not happen if you flash the `opendtu-generic_esp32.bin` via the web interface)

But due to the fact that you erased everything you have to upload the device profile again. Please see https://github.com/tbnobody/OpenDTU/wiki/Device-Profiles for a general documentation and https://github.com/tbnobody/OpenDTU/blob/master/docs/DeviceProfiles/blinkyparts_esp32.json as the correct device profile.

**Reply by JorisZzz** [reply]
That did the trick, thanks for your help!
